example.yaml

```diff
@@ -196,24 +196,24 @@ trace_lightstep_num_clients: 1
 
 # == Kafka ==
 
-# Comma-delimited list of brokers suitable for Sarama's [NewSyncProducer](https://godoc.org/github.com/Shopify/sarama#NewSyncProducer)
-kafka_broker: ""
+# Comma-delimited list of brokers suitable for Sarama's [NewAsyncProducer](https://godoc.org/github.com/Shopify/sarama#NewSyncProducer)
```

Comment:
probably should change the url to point to the async producer section of the docs..

Reply:
Good catch! I thought I got that one. (Also, sorry for the delays, as you can see I'm kinda romping around changing a lot of things. Do let me know if something in here seems problematic for your use case!)